src/main/java/org/logstash/tcp/InputLoop.java

```diff
@@ -188,6 +190,8 @@ private static final class DecoderAdapter extends ChannelInboundHandlerAdapter {
              */
             private final Logger logger;
 
+            private final List<String> silentErrs = Arrays.asList("Connection reset by peer", "Connection reset");
```

Comment:
```suggestion
            private final String CONNECTION_RESET_ERROR_MSG = "Connection reset";
```

Reply:
You are right! It is just "Connection reset".